GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -262,14 +271,16 @@ private bool TryCheckIfUpgradeAvailable(out Version newestVersion, out string co
 
                 if (newestVersion == null)
                 {
-                    consoleError = "Upgrade is not available.";
-                    this.tracer.RelatedInfo("No new upgrade releases available");
-                    return false;
+                    consoleError = null;
+                    this.output.WriteLine(message);
+                    this.tracer.RelatedInfo($"No new upgrade releases available. {message}");
+                    return true;
                 }
 
                 activity.RelatedInfo("Successfully checked for new release. {0}", newestVersion);
             }
 
+            consoleError = null;

```

Comment:
Minor style comment: For consistency you could set `consoleError` to `null` at the start of the method (like you do for `newestVersion`.  This would also allow you to remove ` consoleError = null;` on line 274

Reply:
Disregard, see discussion regarding this style in the comment for `TryRunUpgrade`

cc: @derrickstolee